protected/humhub/modules/content/components/ContentActiveRecord.php

```diff
@@ -269,6 +283,31 @@ public function getContentDescription()
         return "";
     }
 
+    /**
+     * Returns the $createPermission settings interpretable by an PermissionManager instance.
+     *
+     * @since 1.13.0
+     * @see ContentActiveRecord::$createPermission
+     * @return null|object|string
+     */
+    public function getCreatePermission()
+    {
+        return $this->hasCreatePermission()
+            ? $this->getPermissionValue($this->createPermission)
+            : null;
+    }
+
+    /**
+     * Determines weather or not this records has an additional createPermission set.
```

Comment:
`Determines whether or not the record has an additional createPermission set.`

Reply:
Fixed here https://github.com/humhub/humhub/commit/ddeaa105964d824f0a6663b6baa815451ba38c02#diff-5fea8ec3ab977d4e8e1db5cf83b2509d7cdccff1e612ca16efb88c2a37e18793R301